src/state/store/test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-unused-vars */
+import { createStore as createStoreMocked2, applyMiddleware as applyMiddlewareMocked } from 'redux';
+import createSagaMiddlewareMocked from 'redux-saga';
+
+import { store } from './index';
+
+// const mockCreateStore = jest.fn();
+// jest.mock('redux', () => ({ createStore: mockCreateStore }));
+jest.mock('redux', () => ({ createStore: jest.fn() }));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['jest' is not defined.](https://app.codacy.com/manual/iagoregiani/verificador-associacao/pullRequest?prid=5497764)